README.md

```diff
@@ -61,11 +61,11 @@ Now you are good to go!
 ### repo approve
 
 ```sh
-$ repo approve [regex]
+$ repo approve [--title title]
 ```
 
-Iterates over all open pull requests matching `regex` (all PRs if
-no regex is given) in all configured repositories.
+Iterates over all open pull requests matching `title` (all PRs if
+no regex for title is given) in all configured repositories.
```

Comment:
This is now inaccurate - it will not process all PRs.

Reply:
Sorry for the unclear statement, the case I wanna cover is that the command 
`approve --branch finally-drop-node8 --title`  or `approve --branch finally-drop-node8`(if no regex for title is given or title option is not specified) then we will match all PRs with the branch name ` finally-drop-node8`. 

`const regex = new RegExp((cli.flags.title as string) || '.*');` from `asyncPrIterator.ts` will ensure this.